### ISSUE TITLE
HDMI-CEC on Samsung TV

### ISSUE THREAD

**Issue by Braetzelbush** [maxnet/berryboot#402]
Hi,
I had Berryboot version 20161118-pi2-pi3 on my Raspberry and the CEC with my Samsung TV works. After an upgrade to the new version 20170509-pi2-pi3 the TV still sees the Raspberry as a CEC device, but no button on the remote is working.
When I go back to the old version, it works again.

Hardware:
Raspberry Pi 3 Model B
Samsung TV Model LE40A856S1M

**Comment by baferdi** [comment]
Hi,
I have exactly the same problem.
Only the tv is not Samsung but Sony w755c Android TV (2015).

**Comment by maxnet** [comment]
Fixed in 20170527

**Comment by Braetzelbush** [comment]
I testet 20170527 its still the same
TV shows the Pi as a Anynet Device but no one of the buttons on the remote is working.
i.e. when i press the red button the TV shows "not available"

**Comment by maxnet** [comment]
>TV shows the Pi as a Anynet Device but no one of the buttons on the remote is working.

Does show it as Berryboot and not as something else?
Also make sure "source" is set to HDMI-CEC, not HDMI/DVI

So NOT like this:

![20170530_151854](https://cloud.githubusercontent.com/assets/1123701/26600353/0fa239d8-457c-11e7-8ca5-41528596662f.jpg)

**Comment by Braetzelbush** [comment]
it was looking like this
![img_20170530_213519 - kopie](https://cloud.githubusercontent.com/assets/28927111/26601987/43b454da-4582-11e7-9e59-59c5445a777d.jpg)
this was the not available message
![img_20170530_213711 - kopie](https://cloud.githubusercontent.com/assets/28927111/26602005/4d9a882a-4582-11e7-8d00-827c219fe1b0.jpg)


but now comes the funny part
after another anynet-scan the pi only was showed as "recorder" i cant connect to the pi
so i decide to disconnect the tv from the electricity for a few seconds and try again

I started the TV after that, I started the Pi, now it works. When I switch sources now or push the "tools" button, then the buttons stoped working in Berryboot but still worked in openelec. After a reboot it works again.

But when the pi wakes up my TV from standbymode, then I have the old problem. Buttons dont work in Berrybot and dont work in openelec.
This stays until I disconnect the TV from the power again.

Has my TV a bad implemtation of anynet?
